SlackPirate.py

```diff
@@ -8,6 +8,7 @@
 import colorama
 import requests
 import termcolor
+from constants import *
```

Comment:
You are only using a single item from the module, just import it directly

Reply:
Fixed!